services/static-webserver/Dockerfile

```diff
@@ -10,21 +10,21 @@ ENV SC_USER_ID=8004 \
 
 RUN adduser -D -u ${SC_USER_ID} -s /bin/sh -h /home/${SC_USER_NAME} ${SC_USER_NAME}
 
-# front-end client last (image name is the path to the Dockerfile)
-COPY --from=client/tools/qooxdoo-kit/builder:latest --chown=${SC_USER_NAME}:${SC_USER_NAME} \
-  /project/build-output "/static-content"
-
 # changing ownership of static-web-server files
-RUN chown -R ${SC_USER_NAME}:${SC_USER_NAME} /entrypoint.sh && \
-  chown -R ${SC_USER_NAME}:${SC_USER_NAME} /usr/local/bin/static-web-server && \
-  chown -R ${SC_USER_NAME}:${SC_USER_NAME} /public
+RUN chown -R "${SC_USER_NAME}:${SC_USER_NAME}" /entrypoint.sh && \
+  chown -R "${SC_USER_NAME}:${SC_USER_NAME}" /usr/local/bin/static-web-server && \
+  chown -R "${SC_USER_NAME}:${SC_USER_NAME}" /public
+
 
 USER ${SC_USER_NAME}
 
 FROM base as build
+# front-end client last (image name is the path to the Dockerfile)
+COPY --from=client/tools/qooxdoo-kit/builder:latest --chown=${SC_USER_NAME}:${SC_USER_NAME} \
+  /project/build-output "/static-content"
```

Comment:
MINOR: why quotes in one and not the other path? Not required in this case, right?

Reply:
to be honnest I did not bother more with this. I think this needs a more thorough overhaul which would also remove the need of building a separate client before the whole build goes. we would gain 1 minute over the build process.

Reply:
no reason for that, escapes can be removed